src/viking.js

```diff
@@ -1,11 +1,71 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength) {
+        this.health= health;
+        this.strength =strength;
+    }
+    attack() {
+    return  this.strength;
+    }
+    receiveDamage(theDamage){
+        this.health -= theDamage;
+    }
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier{
+    constructor(name, health, strength) {
+        super(health, strength)
+        this.name = name
+    }
+    attack() {
```

Comment:
Since `Viking` is extending the class `Soldier` you don't need to declare the `attack()` method here again because it inherits it from the `Soldier:`

```
class Viking extends Soldier {
  constructor(name, health, strength) {
    super(health, strength);
    this.name = name;
  }
  receiveDamage(theDamage) {
    this.health -= theDamage;
    if (this.health > 0)
      return `${this.name} has received ${theDamage} points of damage`;
    else return `${this.name} has died in act of combat`;
  }
  battleCry() {
    return "Odin Owns You All!";
  }
}
```